src/views/homepage/index.tsx

```diff
@@ -89,4 +90,6 @@ function HomePageView({ content }: HomePageProps): ReactElement {
 }
 
 HomePageView.layout = BaseNewLayout
+HomePageView.theme = GlobalThemeOption.light
```

Comment:
Enforces 'light' at the homepage always. Any page can set this if we want to force themes later on.

Reply:
Nice, this is a neat approach for overrides!